README.md

```diff
@@ -208,6 +208,33 @@ int total = new LengthOf(
 ).intValue();
 ```
 
+To get a set of unique elements by providing varargs:
+
+```java
+final Set<String> unique = new SetOf<String>(
+    "one",
+    "two",
+    "one",
+    "three"
+);
+```
+
+To get a set of unique elements from existing iterable:
```

Comment:
@fanifieiev a phrase `set of unique elements` could be a little bit excessive one, because the definition of [Set](https://en.wikipedia.org/wiki/Set_(abstract_data_type)) already implies that:

> set is an abstract data type that can store **unique** values

@fanifieiev what do you think?

Reply:
@iakunin Will fix that.